cardano_node_tests/utils/tx_view.py

```diff
@@ -174,7 +172,7 @@ def check_tx_view(  # noqa: C901
     tx_raw_len_certs = len(set(tx_raw_output.tx_files.certificate_files)) + len(
         tx_raw_output.complex_certs
     )
-    loaded_len_certs = len(set(tx_loaded.get("certificates") or ()))
+    loaded_len_certs = len(tx_loaded.get("certificates") or [])
```

Comment:
I think this check will not work if one certificate is passed multiple times (that is allowed).